app/templates/hobbies.html

```diff
@@ -2,6 +2,11 @@
 
 {% block title %}Hobbies{% endblock %}
 
+{% block header %}
+{{ super() }}
+<link rel="stylesheet" href="{{ url_for('static', filename='styles/hobbies.css') }}">
```

Comment:
`{{ super() }}` is to import the parent header in order to append to it, rather than override.... Is that right?